modules/engine.py

```diff
@@ -22,6 +23,14 @@ def main_loop(self):
                 play = False
             elif command == "help":
                 self.display_help()
+            elif command == "begin":
+                game = modules.world.Room('resources\Game_Setup.csv')
+                game.get_room_data()
+                print ("Let us begin!")
```

Comment:
Also there is an engine.greet() method that this might make more sense to be in.